docs/source/tutorial/representations.rst

```diff
@@ -30,6 +32,10 @@ relations (including inverse relations) as tokens.
 
 .. seealso:: https://towardsdatascience.com/nodepiece-tokenizing-knowledge-graphs-6dd2b91847aa
 
+Bases
+~~~~~
+.. todo:: write something about :class:`pykeen.nn.emb.LowRankEmbeddingRepresentation`
```

Comment:
Add some high-level docs please! Most importantly:

1. Why does this work
2. How is it different from NodePiece
3. Who is the target audience for knowing this exists (is this a tool an end-user should use from the pipeline, or just a component for developers of new models?)